transport/tlscommon/ca_pinning_test.go

```diff
@@ -360,10 +360,19 @@ func genSignedCert(
 	commonName string,
 	dnsNames []string,
 	ips []net.IP,
+	expired bool,
 ) (tls.Certificate, error) {
 	if commonName == "" {
 		commonName = "You know, for search"
 	}
+
+	notBefore := time.Now()
+	notAfter := notBefore.Add(time.Hour)
```

Comment:
I would suggest more than 1 hour, just for insurance around daylight savings changes.  Yes, the golang time library should get that right, but I really don't want to debug some edge case if it doesn't.

Reply:
Good catch!